macros/B2GMain.ejs

```diff
@@ -3,6 +3,7 @@
 
 var prevStr = mdn.localString({
     "en-US": "« Back to Boot to Gecko",
+    "zh-CN": "« 返回 Boot to Gecko",
```

Comment:
This doesn't break the page, tested on a copy of [B2G_OS/Automated_testing](https://developer.mozilla.org/en-US/docs/Archive/B2G_OS/Automated_testing).  However, the macro has been altered to create no output, so the translation is unnecessary.